spec/system/support/feedbacks_spec.rb

```diff
@@ -0,0 +1,129 @@
+require "rails_helper"
+
+RSpec.describe "Support console feedback view", service: :support do
+  include ActionView::Helpers::TextHelper
+  let(:user) { create(:user, :admin) }
+
+  before { given_i_am_authenticated }
+
+  context "when navigating to the feedback page via the support console" do
+    before do
+      when_i_navigate_to_the_feedback_page
+    end
+
+    scenario "user sees feedback table" do
+      then_i_see_feedback_table
+      then_i_see_recent_feedback_entries
+    end
+
+    scenario "check for backlink presence and navigation on feedback page" do
+      then_i_see_backlink_to_support_homepage
+      click_link_or_button "Back"
+      then_i_am_on_the_support_homepage
+    end
+  end
+
+  context "with more than one page of feedback" do
+    before do
+      create_list(:feedback, 15)
+      when_i_visit_the_feedback_page
+    end
+
+    scenario "user sees pagination controls and can navigate to next page" do
+      then_i_see_first_page_of_feedback_with_pagination
+
+      click_link "Next"
+
+      then_i_see_second_page_of_feedback_with_pagination
+    end
+  end
+
+  context "when a feedback experience is over the truncation limit" do
+    let!(:long_feedback) do
+      create(:feedback, experience: "This is a very long feedback message designed to be more than 100 characters long so that it can trigger the truncation logic and display the 'View full' link.")
+    end
+
+    before do
+      when_i_visit_the_feedback_page
+    end
+
+    scenario "displays truncated experience with 'view full' link, leading user to feedback details show page with bespoke backlink" do
+      then_i_see_truncated_feedback_with_link(long_feedback)
+
+      click_link "View full"
```

Comment:
can we move this to a method too? e.g when_i_click_to_see_the_full_experience (or a similar name)?